app/api/margatsni/v1/entities/post.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Margatsni
+  module V1
+    module Entities
+      class Post < Grape::Entity
+        format_with(:european_timestamp) do |date|
+          date.strftime('%d.%m.%y %H:%M')
+        end
+
+        expose :id
+        expose :body
+        expose :user do |instance|
+          Margatsni::V1::Entities::User.represent(instance.user, except: %i[email])
```

Comment:
I'm not sure, but seems like you can use simply `User.represent(instance.user, except: %i[email])` here, as you are locating in the same module.